Realm/Tests/NotificationTests.m

```diff
@@ -1256,4 +1270,129 @@ - (void)testDiffedUpdatesOnlyNotifyForPropertiesWhichActuallyChanged {
     [token invalidate];
 }
 
+#pragma mark - Object Notification Key Path Filtering
+
+-(void) testModifyObservedKeyPathLocally {
```

Comment:
Another nitpick, but the format differs from the other functions. Should probably be
`- (void)testModifyObservedKeyPathLocally` instead of 
`-(void) testModifyObservedKeyPathLocally` for all functions from here on.

Reply:
Should be resolved [here](https://github.com/realm/realm-cocoa/pull/7249/commits/9465b5d3b85471e46621fa77d7c62c313ad29127)